docs/in-depth-guide-scraper-functions.md

```diff
@@ -316,7 +315,7 @@ Returns a `list` of reviews about the recipe from the website. Each review is a
 
 ### `equipment() -> List[str] | None`
 
-Returns a list of cooking equipment needed for provided recipe.
+Returns a list of cooking equipment needed for the provided recipe.
```

Comment:
From reading the other recipe method docstrings, we've more-or-less standardi(s|z)ed on using the definitive article ('the') in the explainer text - and I think that means we can/should drop the word 'provided' here.

```suggestion
Returns a list of cooking equipment needed for the recipe.
```